insights/tests/test_remote_resource.py

```diff
@@ -24,10 +24,20 @@ def test_get_remote_resource_not_found(self):
         rtn = rr.get(url)
         assert rtn.content == NOT_FOUND
 
+    # Test disable RemoteResource
+    def test_get_disable_remote_resource(self):
+        RemoteResource.Disable_Remote_Resource_Access = True
```

Comment:
After implemented the configurable option, the test of this flag can be implemented via reading a testing `manifest`